Xargo.toml

```diff
@@ -1,7 +1,8 @@
 [dependencies]
-collections = {}
-rand = {}
+core = {}
+alloc = {}
 
 [dependencies.compiler_builtins]
 features = ["mem"]
+git = "https://github.com/rust-lang-nursery/compiler-builtins"
```

Comment:
why should we switch to the git version? rust-src contains a snapshot of that git repo. Is there some version mismatch problem?

Reply:
I wasn't aware that compiler-builtins was distributed with rust-src. Should it be removed from the other Xargo.toml files too?

Reply:
stupid question - I guess it makes sense for the Xargo.test.toml file since it there are a lot of "supported" platforms